static/src/components/GameScreen.js

```diff
@@ -224,17 +332,22 @@ export class GameScreen extends Component {
         if (!started) {
             this.playersLogged();
         }
-
-        if (started && game) {
-            if (game.game.age) {
-                document.title = `Age: ${game.game.age} Round: ${game.game.round}`;
-                nextWonderLevel = game.player.wonder_level + 1;
-                if (nextWonderLevel > game.player.max_wonder) {
-                  canPlayWonder = false;
-                }
+        if (started && game.game.age) {
```

Comment:
You've overwritten Jacobs change on the game check. If you access game.game.age and game is null then this will through a null reference exception

Reply:
I hear ya... Changed